src/containers/ConceptPage/ConceptForm/ConceptForm.tsx

```diff
@@ -222,19 +224,20 @@ const ConceptForm = ({
               >
                 <CopyrightFieldGroup enableLicenseNA={true} />
               </FormAccordion>
-              <FormAccordion
-                id="metadata"
-                title={t("form.metadataSection")}
-                hasError={!!(errors.tags || errors.metaImageAlt || errors.subjects)}
-              >
-                <ConceptMetaData subjects={subjects} inModal={inModal} language={language} />
-              </FormAccordion>
-              <FormAccordion id="articles" title={t("form.articleSection")} hasError={!!errors.articles}>
-                <ConceptArticles />
-              </FormAccordion>
-              <FormAccordion id="versionNotes" title={t("form.workflowSection")} hasError={false}>
-                <SimpleVersionPanel editorNotes={concept?.editorNotes} />
-              </FormAccordion>
+              {!inModal && (
+                <FormAccordion
+                  id="metadata"
+                  title={t("form.metadataSection")}
+                  hasError={!!(errors.tags || errors.metaImageAlt || errors.subjects)}
+                >
+                  <ConceptMetaData subjects={subjects} inModal={inModal} language={language} />
+                </FormAccordion>
+              )}
```

Comment:
Hvorfor er ikke dette lov i dialog?

Reply:
For å forenkle opprettelse av forklaring.